packages/pluggableWidgets/datagrid-web/src/components/__tests__/Table.spec.tsx

```diff
@@ -52,7 +52,6 @@ describe("Table", () => {
                 header: "Test",
                 hasWidgets: false,
                 sortable: false,
-                filterable: "custom" as const,
```

Comment:
Why not set it to true here and check out the structure?

Reply:
because filterable is not a valid property here. I just removed the invalid one :)